core/src/main/boilerplate/pureconfig/ProductReaders.scala.template

```diff
@@ -26,8 +34,16 @@ trait ProductReaders {
         a1Result
       }
   }
-
-  [2..22#final def forProduct1[B, [#A0#]]([#keyA0: String#])(f: ([#A0#]) => B)(implicit
+  [2..22#
+  /**
```

Comment:
Nitpick: Can you move this to the line above and add an extra line before the `[2..22#...`? Just so that the methods only have a blank line between them in the generated code. The same thing applies for `ProductWriters`.

Reply:
Sure, fixed in 3076681.